extension_scripts/install_mellanoxofed.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+distro=`./../common/extract_distro.sh`
+if [ $? -eq 0 ]
+then
+    echo "Detected distro: ${distro}"
+else
+    echo "*** Error - invalid distro!"
+    exit -1
+fi
+
+if [[ $distro == "CentOS Linux 7.6.1810" ]]
```

Comment:
I'd imagine the minor version (1810) would have to be stripped off, no?
Same everywhere in this file and tests/run_tests.sh